test/dynamo/test_logging.py

```diff
@@ -132,26 +132,21 @@ def test_dynamo_debug_default_off_artifacts(self, records):
         self.assertEqual(len([r for r in records if ".__bytecode" in r.name]), 0)
         self.assertEqual(len([r for r in records if ".__output_code" in r.name]), 0)
 
-    @make_logging_test()
+    @make_logging_test(graph_breaks=True)
```

Comment:
not sure whether there's a better way to test this now that we can't really error from user code